Gemfile.lock

```diff
@@ -266,47 +266,48 @@ GEM
       minitest (>= 5.1)
       tzinfo (~> 2.0)
       zeitwerk (~> 2.3)
-    addressable (2.8.4)
+    addressable (2.8.6)
       public_suffix (>= 2.0.2, < 6.0)
-    airbrake (11.0.1)
+    airbrake (11.0.3)
       airbrake-ruby (~> 5.1)
-    airbrake-ruby (5.2.0)
+    airbrake-ruby (5.2.1)
       rbtree3 (~> 0.5)
     ansible (0.2.2)
     ar_multi_threaded_transactional_tests (0.5.0)
       activerecord (>= 4.2.0, < 6.2.0)
-    ast (2.4.1)
+    ast (2.4.2)
     attr_encrypted (3.1.0)
       encryptor (~> 3.0.0)
-    audited (5.4.2)
+    audited (5.4.3)
       activerecord (>= 5.0, < 7.2)
       request_store (~> 1.2)
-    autoprefixer-rails (9.4.8)
+    autoprefixer-rails (9.8.6.5)
       execjs
-    awesome_print (1.6.1)
-    aws-eventstream (1.2.0)
-    aws-partitions (1.329.0)
-    aws-sdk-core (3.100.0)
-      aws-eventstream (~> 1, >= 1.0.2)
-      aws-partitions (~> 1, >= 1.239.0)
+    awesome_print (1.9.2)
+    aws-eventstream (1.3.0)
```

Comment:
### [aws-eventstream](https://github.com/aws/aws-sdk-ruby)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.2.0 → 1.3.0

[_[change-log](https://github.com/aws/aws-sdk-ruby/tree/version-3/gems/aws-eventstream/CHANGELOG.md), [source-code](https://github.com/aws/aws-sdk-ruby/tree/version-3/gems/aws-eventstream), [gem-diff](https://my.diffend.io/gems/aws-eventstream/1.2.0/1.3.0)_]





<details>
<summary>Commits</summary>

A change of **21** commits. See the full changes on [the compare page](https://github.com/aws/aws-sdk-ruby/compare/1.2.0...1.3.0).

These are the first 10 commits:
- (90f8b27) [S3 client should send optional content type on copy object](https://github.com/aws/aws-sdk-ruby/commit/90f8b27a7b0be6ed7fae44d762c5e7154afad35c)
- (43e872b) [S3 object should be able to change content type on copy from](https://github.com/aws/aws-sdk-ruby/commit/43e872be2ac542bef6d431307ba7bf7e456127e0)
- (d288c8d) [Version 1.2.1 of the AWS SDK for Ruby](https://github.com/aws/aws-sdk-ruby/commit/d288c8df3d842017b91f4346a42c0a3458ae1337)
- (216fa48) [domain scope to facilitate sharding](https://github.com/aws/aws-sdk-ruby/commit/216fa4836cfc7afef6084e747e9206f2df6452a2)
- (b6ee4ad) [Version 1.2.2 of the AWS SDK for Ruby](https://github.com/aws/aws-sdk-ruby/commit/b6ee4ad5027403e0813b5c4ce85c7a5305c6a100)
- (9e98d9c) [Added documentation for part_number option of MultipartUploa…](https://github.com/aws/aws-sdk-ruby/commit/9e98d9c4a86956877a71eb9d2338e4c9121876cd)
- (c182b0a) [Fetch console output for instance.](https://github.com/aws/aws-sdk-ruby/commit/c182b0afda46124aea7d8899261826a331427786)
- (22bc7e8) [Merge commit '9e98d9c4a86956877a71eb9d2338e4c9121876cd'](https://github.com/aws/aws-sdk-ruby/commit/22bc7e8209824c73e61658dad8b10166a59c962f)
- (ddbd83f) [Version 1.2.3 of the AWS SDK for Ruby](https://github.com/aws/aws-sdk-ruby/commit/ddbd83fa5acb83d328e6fdeb2ccd75e1affdd2ee)
- (543cbe7) [Add sms support for topic subscriptions](https://github.com/aws/aws-sdk-ruby/commit/543cbe7817f783fe64254de68ddb511de88e3f75)


</details>